node/builder.go

```diff
@@ -245,22 +251,20 @@ func Online() Option {
 		// common
 		Override(new(*slashfilter.SlashFilter), modules.NewSlashFilter),
 
-		// Full node
-
-		ApplyIf(isType(repo.FullNode),
+		// Full node or lite node
+		ApplyIf(isFullOrLiteNode,
```

Comment:
We should be able to move a fair bit of things from here to only full node after implementing more things on top of lotus-gateway